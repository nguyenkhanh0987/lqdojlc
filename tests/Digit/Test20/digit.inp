100
137832563 832016175
525553041 861963746
344660920 841456465
835964885 852498953
104452183 552392285
599675381 891326414
695560820 775969845
690339058 942617784
502899005 879637093
148253274 443271810
556548307 855686924
366271402 578045681
968132354 995187825
630781341 785619804
55019554 425803011
647857095 790705527
104160469 431215731
201473817 856322236
80272690 876190136
271756944 750164229
724709191 818744355
351345989 581230166
519130240 861598410
887970679 903921813
229450579 362417108
771045705 958546783
854569796 952516159
38102891 588697941
132513233 233994361
397549192 638069661
984785688 995356115
559607923 706310456
949145251 986613144
803049618 925304150
762138786 787689845
929214582 945052099
744947779 955341880
580058826 584313394
346154507 458906892
149945190 996726947
976272499 998908455
453260195 491673662
48855798 747020395
497087585 699562936
922434906 985820397
742814781 811391548
843061788 843695825
379403026 938840107
963208822 964212373
917531681 982290724
958010872 965893570
97466477 350985024
274779797 490344403
633049842 995482558
722855550 884290606
395136083 400109963
737636186 782477354
784361477 799255593
196868306 856089834
836554529 926581356
463071321 693388829
74053266 597608764
463406372 566347608
762974892 847008046
441360480 897521715
22911879 286896025
34293756 591129512
735648022 957708018
670861987 891786599
392669691 910821218
729290056 871347859
492627152 868644344
409290663 432804529
719481902 833373338
861669917 873418172
554747127 926237042
635527680 834371557
499953864 884963926
842425290 868409769
243684532 422953498
724534114 867043721
71603518 134176967
4872792 475017121
234156672 395024789
393456569 877878943
134362501 499786375
949135430 993461331
850213911 858285976
787979548 815762308
838282536 913121159
58109817 996719188
841621335 975630286
604837970 748267289
107447502 637712140
934057321 997737854
666768261 998207765
216380190 933185023
544618055 577968008
687230662 953559792
973480734 994913852
