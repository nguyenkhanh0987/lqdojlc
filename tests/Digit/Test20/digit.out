92258675
32089643
56521626
966993
69094578
25045403
5924805
18935402
35060664
48851690
28526046
28601512
2840856
15308544
65521798
14432234
53733630
79457618
106300764
60341080
8154563
31760847
32585131
1649879
21739705
13388787
6588414
90420075
18896469
28042065
1216459
17397799
3077821
7733797
3151006
749314
15625601
1010688
16290241
102208174
2383110
6722712
103089289
22451510
4009985
6885447
30935
57378085
87784
3735109
623161
42700784
32876564
29952272
13105365
951966
4459924
2131211
80359721
6603102
28063673
84138943
13022721
6703257
47903331
51088900
91496668
16372299
19569136
53423739
11324204
35715250
2090629
9164554
1176940
34088894
18077780
36198064
1893363
29123213
11115684
12384379
80906246
27811524
50260513
60893702
3887073
670795
2217046
6228275
120314576
9241756
12329290
78631107
4929147
26903287
83532714
4475759
19867355
2293920
